shell/platform/embedder/embedder.cc

```diff
@@ -563,6 +563,15 @@ InferVulkanPlatformViewCreationCallback(
   auto vk_instance = static_cast<VkInstance>(config->vulkan.instance);
   auto proc_addr =
       vulkan_get_instance_proc_address(vk_instance, "vkGetInstanceProcAddr");
+  // Previously we would query "GetInstanceProcAddr" (note the lack of "vk"
+  // prefix) which is incorrect, but we want to avoid breaking code in the
+  // wild, so try the old name also.
+  //
+  // See https://github.com/flutter/flutter/issues/118956 for details
```

Comment:
This comment should be the PR description; if you want a comment here it should just explain what the code does I would phrase it something like this:

```suggestion
  // Past version would only query "GetInstanceProcAddr" (without the "vk"
  // prefix). This is incorrect. To avoid breaking code, we continue to support this
  // incorrect name if the new name is not available.
```